File: src/main/java/frc/utils/DisabledInstantCommand.java
package frc.utils;

import edu.wpi.first.wpilibj2.command.InstantCommand;
import edu.wpi.first.wpilibj2.command.Subsystem;

/*
 * instantCommand that can run while disabled
 */
public class DisabledInstantCommand extends InstantCommand {
    public DisabledInstantCommand(Runnable toRun, Subsystem... requirements) {
        super(toRun, requirements);
    }

    @Override
    public boolean runsWhenDisabled() {
        return true;
    }
}

File: src/main/kotlin/frc/utils/DisabledInstantCommand.kt
package frc.utils

import edu.wpi.first.wpilibj2.command.InstantCommand
import edu.wpi.first.wpilibj2.command.Subsystem

class DisabledInstantCommand(toRun: Runnable, vararg requirements: Subsystem) : InstantCommand(toRun, *requirements) {
    override fun runsWhenDisabled(): Boolean = true
}
